src/Sarif.Multitool/FileWorkItemsCommand.cs

```diff
@@ -5,20 +5,27 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.IO;
-using System.Linq;
 using System.Reflection;
 using System.Text;
 using Microsoft.CodeAnalysis.Sarif.Driver;
-using Microsoft.CodeAnalysis.Sarif.Visitors;
-using Microsoft.CodeAnalysis.Sarif.WorkItemFiling;
+using Microsoft.CodeAnalysis.Sarif.WorkItems;
 using Microsoft.Json.Schema;
 using Microsoft.Json.Schema.Validation;
-using Microsoft.WorkItemFiling;
-using Newtonsoft.Json;
+using Microsoft.WorkItems;
```

Comment:
>WorkItems [](start = 16, length = 9)

Oooh! We lay claim to the second-level namespace! How exciting :-)

Reply:
Perhaps we should try something like this? 
<Company>.(<Product>|<Technology>)[.<Feature>][.<Subnamespace>]

For example: Microsoft.Office.PowerPoint. 

Maybe: Microsoft.SecurityTooling.WorkItems?



---
In reply to: [381008882](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381008882) [](ancestors = 381008882)

Reply:
I agree with your appeal to the Guidelines; I'm not sure "SecurityTooling" counts as a "product|technology", and even if it did, I don't think "work item filing" would fall under it.

I actually do think "WorkItems" would be a fine second-level namespace; the problem at MS is that there's no registry to allow you to claim it. It's the Wild West out there.

---
In reply to: [381424774](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381424774) [](ancestors = 381424774,381008882)

Reply:
Microsoft.WorkItems is arguably correct because we aren't building a work items API that's specific to security tooling or any other product. We are not a part of the BCL, however, so we are not valid for the System.Subsystem naming approach.


---
In reply to: [381476992](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381476992) [](ancestors = 381476992,381424774,381008882)

Reply:
Makes sense

---
In reply to: [381633442](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381633442) [](ancestors = 381633442,381476992,381424774,381008882)